sample/src/androidTest/java/com/schibsted/spain/barista/sample/RadioButtonsReallyFarAwayTest.java

```diff
@@ -62,4 +66,10 @@ public void checkRadioButtonsByPosition_withLastOption() {
     clickRadioButtonPosition(R.id.radiogroup_really_far_away, 1);
     assertDisplayed("" + R.id.last_item_really_far_away);
   }
+
+  @Test
+  public void farAwayCheckRadioButtonContainsText() {
```

Comment:
I don't think this test should be part of this class. 
The `assertContains` method belongs to the `AssertionsTest`. I don't see the need to specifically add it to RadioButtons test.

Reply:
Done